README.md

```diff
@@ -1,22 +1,37 @@
-# A Discord Rich Presence addon for Guild Wars 2.
-
-https://gw2rpc.info
-
-## Features:
-* Displaying off:
-   + current map, as well as closes point of interest
-   + current raid or fractal boss
-   + character name, race and profession (elite spec)
-   + commander icon if player is currently commanding a squad
-   + active guild tag (needs API key)
-   + time spent on map
+<p align="center"><img alt="Guild Wars 2 RPC" src="https://media.discordapp.net/attachments/908219512271286282/965798895433170954/GW2.png?width=473&height=473" height="76"></p></img>
+
+<h1 align="center">Guild Wars 2 RPC</h1>
+<p align="center">A Discord Rich Presence addon for Guild Wars 2.</p>
+
+<a href="https://gw2rpc.info"><p align="center">Go to website</a>・<a href="https://github.com/Maselkov/GW2RPC/releases">Download Latest</a>・<a href="https://github.com/Maselkov?tab=repositories">Other Projects...</p></a>
+
+---
+<p align="center">You need install <img src="https://api.iconify.design/bi:discord.svg?color=%23f9f9f9" height="12"> <b><a href="https://discord.com/download">Discord Desktop version</b></a> to run the rich presence.</p>
+
+<h2><img src="https://api.iconify.design/ic:baseline-auto-awesome.svg?color=%23ff8cf3" height="20">・Features</h2>
+
+<details markdown='1'><summary>Displaying off</summary>
+
+* current map, as well as closes point of interest
+* current raid or fractal boss
+* character name, race and profession (elite spec)
+* commander icon if player is currently commanding a squad
+* active guild tag (needs API key)
+* time spent on map
 * Automatic raid announcer
 * Automatic update checking
 * Web based registry for maps
 * Configurable settings
 * Supports multiple accounts
----
-## How to install
+</details>
+
+<details markdown='1'><summary>To next features</summary>
```

Comment:
I'd rather not have a rickroll in the official readme. If it helps you, you got me :P